firmware/shared/controls/driver_interface_torque_req_test.cc

```diff
@@ -0,0 +1,30 @@
+// don't need `#pragma once` in a .cc / source file
+
+#include "driver_interface_torque_req.h"  // include the header with your function implementation
+
+#include <iostream>
+
+#include "testing.h"
+
+int main() {
+    // Write your test cases.
+
+    // should return 0
+    ASSERT_CLOSE(ctrl::driver_interface_torque_req(false, false, 2.5), 0);
+
+    // should return accel_ped_pos = 2.5
+    ASSERT_CLOSE(ctrl::driver_interface_torque_req(true, false, 2.5), 2.5);
+
+    // should return accel_ped_pos = 0.0006f
+    ASSERT_CLOSE(ctrl::driver_interface_torque_req(true, false, 0.0006f),
+                 0.0006f);
+
+    // should return accel_ped_pos = 0.4
+    ASSERT_NCLOSE(ctrl::driver_interface_torque_req(false, true, 0.4), 0.5);
```

Comment:
This test is "stronger" if we test for equality with `0.4` rather than inequality with some other value. For example, if your function returned 0.6 instead of 0.4, it would still pass.

You do not need to use an `_NCLOSE` test here if _CLOSE is better suited